app/assets/stylesheets/application.css.scss

```diff
@@ -3,6 +3,10 @@
 
 @import "normalize";
 
-@import "daplog/_base";
-@import "daplog/_blocks";
-@import "daplog/_contexts";
+@import "daplog/base";
+@import "daplog/blocks";
+@import "daplog/contexts";
+
+.2foo{
+  color:red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Colon after property should be followed by one space